### PR TITLE
[connect][sonic-clear] Align the exit code with consutil for line commands

### PR DESCRIPTION
**Pull request by Blueve**
**- What I did**

Align following commands' exit code with consutil commands since there are aliasing commands:
- `connect line`
- `sonic-clear line`

This is a fix for this issue: https://github.com/Azure/sonic-utilities/issues/1241 
The issue will also cause the reverse ssh script couldn't response as expect for console switch feature disabled user(https://github.com/Azure/sonic-buildimage/pull/5438)

**- How I did it**
Bump the inner command's exit code to user.

**- How to verify it**
Installed the script on a Physical DUT and test it.

**- Previous command output (if the output of a command-line utility has changed)**
```
admin@sonic:~$ consutil connect 2
Cannot connect: line [2] has no baud rate
admin@sonic:~$ echo $?
0
```

**- New command output (if the output of a command-line utility has changed)**

```
admin@sonic:~$ consutil connect 2
Cannot connect: line [2] has no baud rate
admin@sonic:~$ echo $?
4
```

### CODE REVIEW

**Comment by lguohan** [comment]
can you list all exit code?

**Comment by Blueve** [comment]
> can you list all exit code?

Sure. We defined them in consutil's lib.py(https://github.com/Azure/sonic-utilities/blob/master/consutil/lib.py)

```
ERR_DISABLE = 1
ERR_CMD = 2
ERR_DEV = 3
ERR_CFG = 4
ERR_BUSY = 5
```

File: connect/main.py [hunk 1]
```diff
@@ -93,7 +96,7 @@ def connect():
 def line(target, devicename):
     """Connect to line LINENUM via serial connection"""
     cmd = "consutil connect {}".format("--devicename " if devicename else "") + str(target)
-    run_command(cmd)
+    sys.exit(run_command(cmd))
```
Comment (lguohan) [inline connect/main.py:99]: why not change line 109 as well?

Comment (Blueve) [reply connect/main.py:99]: This is not console command, I think we need change it in another PR if need.
Currently, I think we only leverage console related command's exit code, I think it is OK to keep other command no change.

Comment (lguohan) [reply connect/main.py:99]: what do you mean 109 is not console command? it is also consutil.

Comment (Blueve) [reply connect/main.py:99]: ahh... i see, I will update it

Comment (Blueve) [reply connect/main.py:99]: This is a undocumented command. I fixed it and updated the document

**Review by lguohan — APPROVED** [review]

**Pull request merged**